projects/packages/sync/composer.json

```diff
@@ -8,6 +8,7 @@
 		"automattic/jetpack-constants": "^1.6",
 		"automattic/jetpack-heartbeat": "^1.3",
 		"automattic/jetpack-options": "^1.11",
+		"automattic/jetpack-password-checker": "0.1.x-dev",
```

Comment:
I think we want to use '^0.1" for the version here. `0.1.0` is the first release of the `jetpack-password-checker` package.

Reply:
Also I think since we're getting that package as a direct dependency of Sync, we should remove the package from Jetpack's dependencies.

Reply:
> Also I think since we're getting that package as a direct dependency of Sync, we should remove the package from Jetpack's dependencies.

Assuming that's the only place the Password Checker is being used. Just be sure to confirm that before removing it directly from Jetpack deps.

Reply:
Fixed.